dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi3.0-appSecurity.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
The javaee version of this autofeature had 1.0 after appSecurity. Did you remove it on purpose?  Should it take on 4.0?

Reply:
Yes. I didn't want to tie it to any version of appSecurity, just that app security is enabled.